src/linagora.esn.calendar/app/app.config.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+require('./core/application-menu-calendar.directive.js');
+
+angular.module('esn.calendar')
+  .config(injectApplicationMenuDirective)
+  .config(registerI18N);
+
+function injectApplicationMenuDirective(dynamicDirectiveServiceProvider) {
+  var dd = new dynamicDirectiveServiceProvider.DynamicDirective(true, 'application-menu-calendar', {priority: 40});
```

Comment:
This is new code so I guess we can use `const` here

Reply:
And I'm OK with other files

Reply:
Well this part of the code will be removed due to https://github.com/OpenPaaS-Suite/esn-frontend-calendar/pull/43.

Reply:
Ok :v